### DISCUSSION TITLE
Should linux users still have passwords?

### DISCUSSION THREAD

**Discussion by messutied** [gravitational/teleport#9997] [Q&A] [answered]
Hi there,

We're setting teleport successfully in order to streamline and secure SSH access to our Linux servers, so far is looking great.

But a question just popped up in my head, now that users are required strict 2FA in order to access teleport, should the Linux users still be password protected?

The idea behind using Teleport for us was to centralize password policies and avoid having to set in place strict password policies on each linux server (we're preparing for ISO27001), which would translate on users having to manage and periodically renew one complex password per server.

Since before we were using normal SSH our Linux users were password protected, is it advisable to remove that password protection with Teleport becoming the only way to access the servers?

What is the recommended good practice here?

**Comment by webvictim** [comment, marked as answer]
Personally, I would leave the `root` password in place in case you need emergency access, but set it to a long, complex value (think `pwgen -s 64 -1`) and store it in a break-glass section of your vault. Other users can likely have their passwords disabled.
